src/js/outreach/apps/verify_app.js

```diff
@@ -20,6 +20,14 @@ export default App.extend({
   beforeStart({ actionId }) {
     return getPatientInfo({ actionId });
   },
+  onFail() {
```

Comment:
I think this needs:

```js
  if (status === 409) {
    this.showAlreadySubmittedView();
    return;
  }

  this.showNotAvailableView();
```
And the only state needed in the verification below is `this.showDialogErrorView();`

Reply:
Yup makes sense, just updated this.